app/src/main/java/com/udacity/asteroidradar/BindingAdapters.kt

```diff
@@ -39,3 +41,13 @@ fun bindTextViewToDisplayVelocity(textView: TextView, number: Double) {
     val context = textView.context
     textView.text = String.format(context.getString(R.string.km_s_unit_format), number)
 }
+
+@BindingAdapter("imageUrl")
+fun bindImage(imgView: ImageView, imgUrl: String?) {
+    imgUrl?.let {
```

Comment:
You could simplify this. You don't need to modify the URL once you get it. Putting it in the let is usually good enough.

```
Glide.with(imgView.context)
        .load(imageUri)
        .into(imgView)
```